tests/integrationv2/tox.ini

```diff
@@ -13,7 +13,8 @@ deps =
     sslyze==5.0.2
     pytest-rerunfailures
 commands =
-    pytest -n 2 --maxfail=1 --reruns=2 --cache-clear -rpfsq \
+   whoami
+    pytest -n 1 --maxfail=1 --reruns=0 --cache-clear -rpfsq \
```

Comment:
The changes to this file shouldn't be committed. This probably explains the CI failures.